packages/create-svelte/templates/default/src/hooks.ts

```diff
@@ -1,33 +1,25 @@
 import cookie from 'cookie';
 import { v4 as uuid } from '@lukeed/uuid';
-import type { GetContext, Handle } from '@sveltejs/kit';
+import type { Handle } from '@sveltejs/kit';
 
-export const getContext: GetContext = (request) => {
+export const handle: Handle = async ({ request, render }) => {
 	const cookies = cookie.parse(request.headers.cookie || '');
+	request.locals.userid = cookies.userid || uuid();
 
-	return {
-		is_new: !cookies.userid,
-		userid: cookies.userid || uuid()
-	};
-};
-
-export const handle: Handle = async ({ request, render }) => {
 	// TODO https://github.com/sveltejs/kit/issues/1046
 	const response = await render({
 		...request,
 		method: (request.query.get('_method') || request.method).toUpperCase()
 	});
 
-	const { is_new, userid } = request.context;
-
-	if (is_new) {
+	if (!cookies.userid) {
```

Comment:
Not really related to this change, but is there a reason not to just mutate the `response` object before returning it, which would be a bit simpler?

Reply:
nope! updated it. feels much nicer.